views/features.erb

```diff
@@ -64,6 +64,7 @@
 			<ul class="list list-bullet">
 				<li>Manage applications with the CloudFoundry command line or API</li>
 				<li>Scale applications up and down in seconds</li>
+				<li>Use private networking to deploy microservices securely</li>
```

Comment:
I worry (very!) slightly about the overloaded term "private networking", and what assumptions Ops folks can place on it.

A non-negligible proportion of folks will (IMHO) assume "my org/app gets its own RFC1918 CIDR", and might make representations to IA based on this assumption.

Reply:
In a sense it (kind of) does - the `apps.internal` route will resolve to something in `10/8`, which is RFC1918 😄.

It's private in the sense that it doesn't get an internet IP, but not in the sense that you get your apps get their own subnet (which some people might assume).

Perhaps `Use private apps to deploy microservices securely` would head off that misunderstanding?

```suggestion
				<li>Use private applications to deploy microservices securely</li>
```

Reply:
I guess we're using the term "microservices" as that is something someone who reads this page as opposed to the tech docs might recognise as a need?

Reply:
Yeah, I think that's the buzzword that people reading the features page would be looking for.

I was discussing with @tnwhitwell whether it might be worth removing the " securely" adverb too, since the feature doesn't magically make your applications secure, it just means they're communicating over private network. Not sure.

Reply:
```suggestion
				<li>Use private networking to deploy microservices that are inaccessible from the public Internet</li>
```